docs/modules/pe.rst

```diff
@@ -560,6 +560,13 @@ Reference
 
         Section name.
 
+    .. c:member:: full_name
+
+        If the name in the section table contains a slash (/) followed by
```

Comment:
Explaining that this is a technique used by some compilers, as you already did in the commit message would be useful in the documentation too. Without this context is hard to figure out why this feature is useful.

Reply:
Changed doc